__tests__/app.test.js

```diff
@@ -113,6 +113,87 @@ describe('GET /api/articles/:article_id', () => {
   });
 });
 
+describe('PATCH /api/articles/:article_id', () => {
+  test('201: Responds with article including updated vote count for given article_id', () => {
+    const testBody = {
+      inc_votes: 1,
+    };
+
+    return request(app)
+      .patch('/api/articles/2')
+      .send(testBody)
+      .expect(201)
```

Comment:
A `200` is fine here - a `201` is the status "created" but a patch is an "update" in the world of CRUD.

Reply:
I've updated the status code to 200 :saluting_face: